rust/connlib/clients/apple/build.rs

```diff
@@ -1,140 +1,147 @@
 // Referenced from https://github.com/chinedufn/swift-bridge/blob/master/examples/rust-binary-calls-swift-package/build.rs
 
-use std::path::PathBuf;
-use walkdir::WalkDir;
+#[cfg(any(target_os = "darwin", target_os = "ios", target_os = "ios-sim"))]
```

Comment:
```suggestion
// TODO: rust-lang/cargo#5220
#[cfg(any(target_os = "darwin", target_os = "ios", target_os = "ios-sim"))]
```
I think when that is closed we should just do platform-specific workspaces.


Also, @thomaseizinger I think the build is failing because you're using `target_os = "darwin"` instead of `target_os = "macos"` ([see available `target_os` options](https://doc.rust-lang.org/reference/conditional-compilation.html#target_os))

Reply:
cc @francesca64